app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -28,6 +28,12 @@ const CONFIRMATION_METHODS = Object.freeze([
   ...SIGNING_METHODS,
 ]);
 
+export type PPOMMiddlewareRequest<
+  Params extends JsonRpcParams = JsonRpcParams,
+> = Required<JsonRpcRequest<Params>> & {
+  securityAlertResponse?: SecurityAlertResponse | undefined;
```

Comment:
Defined `securityAlertResponse` property as both optional and nullable, since both patterns are used in the tests.